src/Components/Button/index.js

```diff
@@ -21,7 +19,12 @@ Button.propTypes = {
   // eslint-disable-next-line react/require-default-props
   style: PropTypes.object,
   action: PropTypes.func.isRequired,
-  variant: PropTypes.string.isRequired,
+  variant: PropTypes.oneOfType([
```

Comment:
nice use of prop types

Reply:
Thank you 😊. I realized much of my previous work did not always include them and plan to take a look through to see where they are missing as it helps Storybook with interactivity